tests/sles4sap/saptune/mr_test.pm

```diff
@@ -268,6 +268,7 @@ sub test_sapconf {
     # Scenario 1: sapconf is running and active with sap-netweaver profile.
     # The test shall show, that a running tuned profile or sapconf is not compromised.
     script_run 'cp /etc/systemd/logind.conf.d/sap.conf{.bak,}';
+    assert_script_run "tuned-adm off" if is_sle('>=15');
```

Comment:
Could you just add a comment for others to understand ***why*** this is needed?

Reply:
Done.